extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -421,7 +421,8 @@ export class SchemaCompareDialog {
 		this.sourceServerDropdown.onValueChanged(async (value) => {
 			if (this.sourceServerDropdown.values.findIndex(x => this.matchesValue(x, value)) === -1) {
 				this.sourceDatabaseDropdown.updateProperties({
-					values: []
+					values: [],
+					value: '  '
```

Comment:
has to be whitespaces?

Reply:
yeah... either real string or spaces... empty string or null is ignore and not updated